### DISCUSSION TITLE
个人知识库

### DISCUSSION THREAD

**Discussion by LotusRootMonkey** [zai-org/ChatGLM3#683] [General]
你好，我想请教一下，我自己本地，有一个我个人的知识库，所有文档都是txt格式的笔记，我想让ChatGLM3对它是全知的，这样其它人就可以通过和ChatGLM3对话，以问答形式，获取知识库的内容，那么我应该如何做呢

**Comment by zRzRzRzRzRzRzR** [comment]
如果开发水平不高可以尝试langchain-chatchat这个项目
